framework/set/resources/rke/aws/createResources.go

```diff
@@ -32,7 +32,10 @@ func CreateAWSResources(file *os.File, newFile *hclwrite.File, tfBlockBody, root
 	createRKEProviderBlock(rootBody)
 	rootBody.AppendNewline()
 
-	instances := []string{rkeServerOne, rkeServerTwo, rkeServerThree}
+	if instances == nil {
```

Comment:
I don't think you need to have this conditional here tbh. This is specifically going to the RKE provider test and nowhere else, so no need.